events-api/src/event_traits.rs

```diff
@@ -13,14 +13,22 @@ pub fn initilize_source_component(comp: &str) {
 
 impl EventMeta {
     /// New event metadata with default values.
-    pub fn new() -> Self {
+    fn new() -> Self {
```

Comment:
Removed pub from new as all the event generation fns can now use from_source to get event meta.